lib/posix/posix_linux_amd64.nim

```diff
@@ -433,8 +433,8 @@ type
                        header: "<sys/socket.h>",
                        pure, final.} = object ## struct sockaddr_storage
     ss_family*: TSa_Family ## Address family.
-    ss_padding: array[128 - sizeof(cshort) - sizeof(culong), char]
-    ss_align: clong
+    ss_padding {.importc: "__ss_padding".}: array[128 - sizeof(cshort) - sizeof(culong), char]
+    ss_align {.importc: "__ss_align".}: clong
```

Comment:
just delete all padding and alignment fields. They are not supposed to be printed. They are not exported, so code won't fail on it. Size and alignment computation is forwarded to the c compiler for imported types.